handler/PostMessageSentHandler.ts

```diff
@@ -59,6 +59,8 @@ export class PostMessageSentHandler {
             if (roomCustomFields && roomCustomFields.isHandedOverFromDialogFlow === true) {
```

Comment:
@bhardwajaditya I realize that we are not actually using the customer_idel_timeout value in the DF config. We have that set to false, and we rely instead on the Visitor Abandonment setting that our enterprise license provides.  Is there any way for you to look at that, or are you blocked because you don't have enterprise?

Reply:
@ear-dev I an blocked due to enterprise license, I will see if I can find something relevant in the issues or discussions